app/src/main/java/com/example/placefinder/dao/IParkDAO.kt

```diff
@@ -6,5 +6,9 @@ import retrofit2.http.GET
 
 interface IParkDAO {
     @GET("JSON/")
-    fun getAllParks() : Call<List<Park>>
+    fun getAllParks() : Call<ArrayList<Park>>
```

Comment:
Keep this as it were.  Per Dependency Inversion, classes should use abstractions (interface List) for variables, instead of concretions (class ArrayList).